py/selenium/webdriver/remote/webdriver.py

```diff
@@ -82,11 +87,11 @@ def __init__(self, command_executor='http://127.0.0.1:4444/wd/hub',
         if type(self.command_executor) is bytes or isinstance(self.command_executor, str):
             self.command_executor = RemoteConnection(command_executor, keep_alive=keep_alive)
         self._is_remote = True
-        self.session_id = None
-        self.capabilities = {}
+        self.session_id = session_id
+        self.capabilities = capabilities
         self.error_handler = ErrorHandler()
         self.start_client()
-        self.start_session(desired_capabilities, browser_profile)
+        self.start_session(desired_capabilities, browser_profile, force_new=False)
```

Comment:
So, I wanted to search around and see what people were doing with this "public" method. And it looks like people are already wrapping RWD and implementing their own start_session method in order to provide this functionality.

example: https://github.com/willcai1984/AerohiveAuto/blob/master/trunk/webui/test/clchen_hm_test.py#L21-L34

So... begs the question, should we really provide something like this in the selenium client library? Or just leave it as an exercise for the user to have non-standard behavior by extending the class themselves?

@AutomatedTester or @andreastt  have any comments there?

I only found one instance on github where someone (mozilla in fact) was calling start_session to start a new session... but I find that use case a little silly and they should just call the constructor instead.

But to help in that manner, rather than having a strange place holder variable, we could make a new method: _start_session that has the logic proposed (and call from the constructor) and leave the existing start_session which would explicitly set session_id to None and then call the new _start_session method.